recipes/rxspencer/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "rxspencer" %}
+{% set version = "3.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/garyhouston/rxspencer/archive/v{{ version }}.tar.gz
+  sha256: 06505066eefe9daa90d9c13469973e6b4d1b52e34a59c465c6a0b8377a2bd7e2
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+
+test:
+  commands:
+    - test -f "${PREFIX}/include/rxspencer/regex.h"                  # [unix]
+    - test -f "${PREFIX}/lib/cmake/rxspencer/RXSpencerConfig.cmake"  # [unix]
+    - test -f "${PREFIX}/lib/librxspencer${SHLIB_EXT}"               # [unix]
+    - if not exist %LIBRARY_INC%\rxspencer\regex.h exit 1            # [win]
+    - if not exist %LIBRARY_INC%\rxspencer\regex.h exit 1            # [win]
+    - if not exist %LIBRARY_LIB%\rxspencer.lib     exit 1            # [win]
+
+about:
+  home: https://garyhouston.github.io/regex
+  license: Custom + CC0-1.0
```

Comment:
@conda-forge/staged-recipes What's the correct expression here for: https://github.com/garyhouston/rxspencer/blob/master/COPYRIGHT

Reply:
https://spdx.org/licenses/Spencer-94.html

Reply:
Thanks!